rca/enquire_to_study/views.py

```diff
@@ -189,14 +189,32 @@ def send_user_email_notification(self, form):
             fail_silently=False,
         )
 
+    def set_session_data(self, form_data, enquiry_submission):
+        # Sets form data into session variable for analytics.
+        self.request.session["enquiry_form_session_data"] = {
+            "event": "form-submission",
+            "formName": "Enquire to study form",
+            "formId": enquiry_submission.id,
+            "countryResidence": form_data["country_of_residence"],
+            "countryCitizen": form_data["country_of_citizenship"],
+            "programme": [i.title for i in form_data["programmes"]],
+            "starting": form_data["start_date"].label,
+            "enquiryType": form_data["enquiry_reason"].reason,
+            "newsletter": True,
```

Comment:
Are they being automatically signed up to a newsletter, or does this maybe need a dynamic value?

Reply:
:medal_military: thanks